assets/src/console/pages/gateway-accounts/CardTypesPage.tsx

```diff
@@ -0,0 +1,122 @@
+import * as React from "react";
+import { Helmet } from "react-helmet";
+import createDecorator from "final-form-focus";
+
+import {
+  PageTitle,
+  P,
+  Checkbox,
+  Button,
+  Form,
+  Hr,
+  Strong,
+  Loader,
+  ErrorAlert
+} from "@pay/web";
+
+import {
+  GatewayAccountFragment,
+  useUpdateGatewayAccountCardTypesMutation,
+  CardTypeBrand,
+  CardTypeType
+} from "../../__generated__/graphql";
+import {
+  useCardTypesQuery,
+  CardTypesQuery
+} from "../../../platform-admin/__generated__/graphql";
+import { CardType } from "../../../auth/__generated__/graphql";
+
+interface FormValues {
+  cardTypes: Array<string>;
+}
+
+const decorators = [createDecorator<FormValues>()];
+
+interface Props {
+  gatewayAccount: GatewayAccountFragment;
+}
+
+const getCardTypeBrand = (cardType: CardType) => {
+  if (cardType.type && cardType.label) {
```

Comment:
you can refactor this to return early so it's easier to follow:

```ts
if (!x) {
  return "";
}
// else
// ...
```

notice that we check for the unhappy path first. that makes the happy path less indented. 

can also just return the value of `label` directly instead of pulling it out into a variable.